src/Components/SignUp/SignUp.js

```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import "./SignUp.css";
 const SignUp = () => {
     return (
          <div className="outer-product-side-bar">
-            <div className="login-container">
+            <div className="signup-container">
                 <div className="login-content">
                     <h2>Sign up</h2>
-                    <label>First Name</label><br></br><br></br>
-                    <input type= "text" className="user-input" placeholder="Enter your Name"/><br></br><br></br>
+                    <label>First Name</label><br/>
+                    <input type= "text" className="user-input" placeholder="Enter your Name"/><br/>
 
```

Comment:
Avoid using br tags as it is an indication of poor semantic HTML. Use elements to seperate out items into seperate visual blocks.

Reply:
> Avoid using br tags as it is an indication of poor semantic HTML. Use elements to seperate out items into seperate visual blocks.

thx to give suggession  I will keep in mind this point